views/partials/header.ejs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Express Daily Planet</title>
+  <link rel="stylesheet" type="text/css" href="/css.css">
+</head>
+<body>
```

Comment:
the header should end here with the opening body tag, the rest should be in your main page. The only extra html I would suggest putting in your header is the nav bar code so it appears on every page.